qa/automation/Alleva 2405/Sign_in_to_app_with_Google_account.feature

```diff
@@ -0,0 +1,15 @@
+Feature: 1.1. Sign in to App with Google account
+
+Scenario: Sign in with Google account granted
+	Given user is on https://qa.ddso-spot.quantori.com/ page
+	When user hits "Sign in with Google" button
+	And "Google account authentication" is granted by Google
```

Comment:
Not an understandable step. And you missed a few steps here: user enters email, clicks on "Next" button, enters password etc